solutions/src/main/java/ru/sberbank/school/task02/Main.java

```diff
@@ -0,0 +1,58 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+
+import java.math.BigDecimal;
+
+
+public class Main {
+    public static void main(String[] args) {
+
+        ExternalQuotesService myExternalQuotesService = new MyExternalQuotesService();
+        ServiceFactoryImpl serviceFactoryImpl = new ServiceFactoryImpl();
+        ExtendedFxConversionService extendedConversionService = serviceFactoryImpl.getExtendedFxConversionService(myExternalQuotesService);
```

Comment:
Line is longer than 120 characters (found 139).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a56380112>*